pmount: add page (#4878)
tldr-pages/tldr: pages/linux/pmount.md

# pmount

> > Mount arbitrary hotpluggable devices as normal user.

Reviewer: This is the best online man page that I found
```suggestion
> Mount arbitrary hotpluggable devices as normal user.
> More information: <https://linux.die.net/man/1/pmount>.
```

Author: Ok, I thought only project homepages would count. Done.

- Mount a device below /media/ (using device as mount point):

> `pmount {{path/to/device}}`

Reviewer: the device is a device unit right? so:
```suggestion
`pmount {{/dev/sdX}}`
```

Author: I'm not sure about this one, since SD cards (which I often pmount) are at `/dev/mmcblk...`.

Author: I'm unsure about this one, since for example SD cards are named differently.

Reviewer: Well, we can use `path/to/file_or_directory` when you can use both as an argument. So, I think we can use:
```suggestion
`pmount {{/dev/sdX_or_mmcblkX}}`
```

Author: `_or_whatever`? I agree that `path/to/device` is bad, but maybe you're too specific here. Is it possible to use a general form like `/dev/device` or `/dev/X`?

Reviewer: :thinking: `dev` is `device` so `/dev/device` is redundant. `/dev/mmcblk` is a helper to guide the reader to find the SD card reader. Maybe we can use:
```suggestion
`pmount {{/dev/to/block/device}}`
```
And it reads similar to `path to file`

Author: This looks fine to me.

> - Mount a device with specified file system type to /media/label:

Reviewer: ```suggestion
- Mount a device with a specific filesystem type to `/media/label`:
```

> `pmount -t {{filesystem}} {{path/to/device}} {{label}}`

Reviewer: Please use long argument format:
```suggestion
`pmount --type {{filesystem}} {{/dev/sdX}} {{label}}`
```

> - Show all mounted removable devices:

Reviewer: ```suggestion
- Display all mounted removable devices:
```

# pmount

> Mount arbitrary hotpluggable devices as normal user.
> More information: <https://linux.die.net/man/1/pmount>.

> - Mount a device below /media/ (using device as mount point):

Reviewer: ```suggestion
- Mount a device below `/media/` (using device as mount point):
```

`pmount {{/dev/to/block/device}}`

- Mount a device with a specific filesystem type to `/media/label`:

`pmount --type {{filesystem}} {{/dev/to/block/device}} {{label}}`

> - Mount a CD-ROM (filesystem type ISO9660) read-only:

Reviewer: ```suggestion
- Mount a CD-ROM (filesystem type ISO9660) in read-only mode:
```

`pmount --type {{iso9660}} --read-only {{/dev/cdrom}}`

> - Mount an NTFS-formatted disk with forced read-write access:

Reviewer: I think this is better:
```suggestion
- Mount an NTFS-formatted disk, forcing read-write access:
```
